src/rgw/rgw_main.cc

```diff
@@ -747,13 +783,15 @@ static int civetweb_callback(struct mg_connection *conn) {
           dout(1) << "DSS INFO: CIVETWEB HEADER NAME: " << name_str << dendl;
```

Comment:
yes, let's print the headers always. We find a nicer way when we revisit logging in the coming days.